rio_viz/templates/template.py

```diff
@@ -0,0 +1,66 @@
+import os
+from typing import Callable, Optional
+
+from starlette.requests import Request
+from starlette.templating import Jinja2Templates, _TemplateResponse
+
+html_templates = Jinja2Templates(directory=os.path.dirname(__file__))
+
+
+def create_simple_template_factory(
+    tilejson: Optional[str] = "_tilejson", info: Optional[str] = "_info"
```

Comment:
@geospatial-jeff do we have to set `Optional` here, because there is a default value I'm not sure we need this?

Reply:
@geospatial-jeff second question, is `_tilejson` indicating the name of the function for the `@self.app.get("/tilejson.json")` ? What kind of black magic this is ?